website/project/model.py

```diff
@@ -2360,6 +2369,14 @@ def rename_node_wiki(self, name, new_name, auth):
                 self.wiki_private_uuids[new_key] = self.wiki_private_uuids[key]
                 del self.wiki_private_uuids[key]
 
+        wiki_updated.send(
+            name,
+            user=auth.user,
+            node=self,
+            action="renamed",
```

Comment:
It may make sense to have separate signals for different actions. That allows more granularity for subscriptions.

This would require you to register a subscription for each of the new signals. Even so, there are multiple ways you can minimize boilerplate and and duplication through, e.g. helper functions.